### PR TITLE
Added magmastream client v4

### PR DESCRIPTION
**Pull request by itsjustnickdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ Version numbers can come in different combinations, depending on the release typ
 ## Client libraries:
 | Client                                                      | Platform | Compatible With                            | Additional Information |
 |-------------------------------------------------------------|----------|--------------------------------------------|------------------------|
+| [Magmastream](https://github.com/Blackfort-Hosting/magmastream)    | Node.js      | **Any**                                    |                        |
```
Comment (topi314) [inline README.md]: the table is ordered by languages. Please format the table accordingly

**Review by topi314 — CHANGES_REQUESTED** [review]

**Review by itsjustnickdev — COMMENTED** [review]
Done

File: README.md (hunk 1 above)
Comment (topi314) [reply README.md]: this is still not aligned

Comment (itsjustnickdev) [reply README.md]: Its aligned on my phone you saw the new commit?

Comment (topi314) [reply README.md]: image

Comment (itsjustnickdev) [reply README.md]: Alphabetical order? Or what do you mean?

Comment (DRSchlaubi) [reply README.md]: 1. There is a missing `|`
2. You need to align all the `|` with the row above

You can read more [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables)

**Comment by itsjustnickdev** [comment]
Good like this?

File: README.md (hunk 1 above)
Comment (topi314) [reply README.md]: image
no

Comment (itsjustnickdev) [reply README.md]: > image
> no

You know we can't change length of the url?
That's the only thing that we can't change so its aligned

Comment (topi314) [reply README.md]: then widen the column?

Comment (itsjustnickdev) [reply README.md]: On all?

Comment (itsjustnickdev) [reply README.md]: You can only see this in the code not in the colom itself

Comment (topi314) [reply README.md]: yes?

Comment (topi314) [reply README.md]: the raw markdown is allowed to be correctly formatted aswell

Comment (itsjustnickdev) [reply README.md]: Good like this if not please talk to @SxMAbel  . I'm going to bed.

**Review by topi314 — APPROVED** [review]

**Pull request merged**